db/models/sign.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var Sign = sequelize.define('Sign', {
+    name: DataTypes.STRING,
```

Comment:
Do we need `id` in the sign model?

Reply:
Sequelize will add this automatically when sync'ing the table (along with created/updated fields). We can be more explicit about this.